src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,58 @@
 package core.basesyntax;
 
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+
 public class WorkWithFile {
+    private static final int INDEX_OF_OPERATION_TYPE_VALUE = 0;
+    private static final int INDEX_OF_AMOUNT_VALUE = 1;
+
     public void getStatistic(String fromFileName, String toFileName) {
+        String[] result = generateResultStatistic(readDataFromFile(fromFileName));
+        File file = new File(toFileName);
+        if (file.exists()) {
+            file.delete();
+        }
+        writeDataToFile(file, result);
+    }
+
+    private Object[] readDataFromFile(String fromFileName) {
```

Comment:
Why Object? Isn't it always String?

Reply:
There were arrays of Objects in all of my methods. I changed it to the String in the newest commit down here. But when I implemented this change, I got a problem in my generateResultStatistic method on 31th line of WorkWithFile.java: I can't split the String (which I got from readDataFromFile method by using Files.readString()) by System.lineSeparator(), it's passing the tests only with splitting by "\n".